error/error.go

```diff
@@ -238,10 +238,34 @@ type ErrAssertionFailed struct {
 	*kvrpcpb.AssertionFailed
 }
 
+// ErrLockOnlyIfExistsNoReturnValue is used when the flag `LockOnlyIfExists` of `LockCtx` is set, but `ReturnValues`` is not.
+type ErrLockOnlyIfExistsNoReturnValue struct {
+	ForUpdateTs uint64
+	LockKey     []byte
+}
+
+// ErrLockOnlyIfExistsNoPrimaryKey is used when the flag `LockOnlyIfExists` of `LockCtx` is set, but primary key of current transaction is not.
+type ErrLockOnlyIfExistsNoPrimaryKey struct {
+	ForUpdateTs uint64
+	LockKey     []byte
+}
+
 func (e *ErrAssertionFailed) Error() string {
 	return fmt.Sprintf("assertion failed { %s }", e.AssertionFailed.String())
 }
 
+func (e *ErrLockOnlyIfExistsNoReturnValue) Error() string {
+	return fmt.Sprintf("LockOnlyIfExists is set for Lock Context, but ReturnValues is not set, "+
```

Comment:
Is `StartTS` needed to be recorded so that related logs could be searched quickly?

Reply:
done